apps/hannk/interpreter/model.cpp

```diff
@@ -5,6 +5,15 @@
 #include <cmath>
 #include <list>
 
+#define WEAK __attribute__((weak))
```

Comment:
This won't compile on Windows where weak linking doesn't exist, so the buildbots here will likely break. At a minimum, wrap these changes in `#ifndef _MSC_VER`.

Reply:
Added compiler guard with HANNK_PROFILER